docs/sources/setup-grafana/configure-security/configure-authentication/azuread/index.md

```diff
@@ -23,17 +23,22 @@ The Azure AD authentication allows you to use an Azure Active Directory tenant a
 
 - [Configure Azure AD OAuth2 authentication](#configure-azure-ad-oauth2-authentication)
   - [Create the Azure AD application](#create-the-azure-ad-application)
+    - [Configure application roles for Grafana in the Azure Portal](#configure-application-roles-for-grafana-in-the-azure-portal)
+    - [Configure Application roles for Grafana in the manifest file](#configure-application-roles-for-grafana-in-the-manifest-file)
```

Comment:
You have application and Application roles - for consistency is should be lowercase or capitalized the same throughout the document.

Reply:
Fixed.